packages/odyssey-react/src/components/Button/Button.theme.ts

```diff
@@ -72,13 +72,13 @@ export const theme: ThemeReducer = (theme) => ({
 
   // Dismiss Variant
   DismissBackgroundColor: "transparent",
-  DismissHoverBorderColor: "transparent",
   DismissTextColor: "inherit",
   DismissDisabledTextColor: theme.ColorNeutralBase,
   DismissLineHeight: 1,
   DismissPaddingBlock: theme.SpaceScale0,
   DismissPaddingInline: theme.SpaceScale0,
-  DismissHoverBackgroundColor: "rgba(255, 255, 255, 0.6)",
+  DismissHoverBackgroundColor: "29, 29, 33",
```

Comment:
Normally, the sass `rgba()` function would convert Hex to RGBA; however, this functionality doesn't work with CSS variables since they aren't compiled.

This is a workable stand-in for now, but we should consider transitioning our tokens to HSL or RGBA since we won't have access to Sass color manipulation.

Reply:
hmmm yea I understand why you want to do this but it is going to be unexpected for folks to theme however as they may not understand it needs to be RGBA formatted for this unique token value. We don't expose that in docs for example. Tracking this as a future improvement will be helpful

Reply:
Fair enough. I'll use a full value until we transition to RGBA or HSLA as the preferred format.